docs/watcher/run-watcher.md

```diff
@@ -0,0 +1,707 @@
+
+This method shows how to install and run Watcher and auxiliary services via Docker tooling.
+
+### 1. Install Dependencies
+
+#### 1.1 Install Docker
+
+```
+sudo curl -sSL https://get.docker.com/ | sh && sudo usermod -aG docker $USER && exit
+```
+
+> - `$USER` - the name of the user with root privileges used on a remote server.
+
+#### 1.2 Install Docker Compose
+
+Make sure to install the latest version of Docker Compose from the [official repository](https://github.com/docker/compose/releases).
+
+```
+sudo curl -L "https://github.com/docker/compose/releases/download/1.25.5/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && sudo chmod +x /usr/local/bin/docker-compose && exit
+```
+
+#### 1.3 Install PostgreSQL
+
+Some Linux servers don't have pre-installed PostgreSQL. You might need to install it manually as follows:
+
+```
+sudo apt update && sudo apt install postgresql postgresql-contrib
+```
+
+#### 1.5 Verify
+
+To verify the installed dependencies, use the following commands:
+
+```
+docker -v && docker-compose -v
+```
+
+Example output:
+```
+Docker version 19.03.9, build 9d988398e7
+docker-compose version 1.25.5, build 8a1c60f6
+```
+
+### 2. Set Up Configuration Files
+
+The Watcher consists of `watcher` and `watcher_info` services. You can run `watcher` separately, however, `watcher_info` relies on the PostgreSQL database where it stores the network's data. All releases and corresponding Docker images (starting from `1.0.1`) can be found in our [`official repository`](https://github.com/omgnetwork/elixir-omg/releases).
+
+#### 2.1 Configure docker-compose-watcher.yml File
+
+Docker Compose allows defining and running multi-container Docker applications. To launch a Watcher with Compose, first, create YAML file that will contain configurations for our services with `nano` or `vim` text editor:
+
+```
+mkdir watcher && cd watcher && nano docker-compose-watcher.yml
+```
+
+Then, copy and paste the [required configs](/files/docker-compose-watcher.yml), save the changes with `ctrl+o` (Linux/Windows) or `control+o` (macOS) and `Enter` to confirm the changes respectively. Then exit the file with `ctrl+x` or `control+x`.
+
+#### 2.2 Configure an Environment File
+
+The YAML file has several values that have to be configured in `.env` file as follows::
+
+```
+echo "WATCHER_IMAGE=$WATCHER_IMAGE
+WATCHER_INFO_IMAGE=$WATCHER_INFO_IMAGE
+ETHEREUM_RPC_URL=$ETHEREUM_RPC_URL
+ETHEREUM_NETWORK=MAINNET
+CHILD_CHAIN_URL=https://childchain.mainnet.v1.omg.network
+AUTHORITY_ADDRESS=0x22405c1782913fb676bc74ef54a60727b0e1026f
+TXHASH_CONTRACT=0x1c29b67acc33eba0d26f52a1e4d26625f52b53e6fbb0a4db915aeb052f7ec849
+CONTRACT_ADDRESS_PLASMA_FRAMEWORK=0x0d4c1222f5e839a911e2053860e45f18921d72ac
+CONTRACT_ADDRESS_ETH_VAULT=0x3eed23ea148d356a72ca695dbce2fceb40a32ce0
+CONTRACT_ADDRESS_ERC20_VAULT=0x070cb1270a4b2ba53c81cef89d0fd584ed4f430b
+CONTRACT_ADDRESS_PAYMENT_EXIT_GAME=0x48d7a6bbc428bca019a560cf3e8ea5364395aad3" > ~/watcher/.env
```

Comment:
These environment variables will not be picked up by docker-compose as they'll stick to whatever is defined in https://github.com/omgnetwork/elixir-omg/blob/master/docker-compose-watcher.yml. Only `$INFURA_API_KEY` will need to be configured.

If you think setting these env vars in an `.env` file is useful for users, let us know, we'll need to figure out how best to incorporate docker-compose's [`env_file`](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option) option so it will pickup the env file.

Reply:
Are you sure they won't be picked up? I don't clone `elixir-omg` repo with this approach. Just create yml file, add env variables and run docker compose. It worked every time I tried it. I run it yesterday actually : )

Reply:
Ah I see it's this file https://github.com/omgnetwork/omgnetwork.github.io/blob/master/files/docker-compose-watcher.yml not the elixir-omg one. Sorry ignore me on this one